tests/test_errors.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from operatorcourier.errors import (
+    OpCourierBadBundle,
+    OpCourierBadYaml,
+    OpCourierBadArtifact,
+    OpCourierQuayError,
+    OpCourierError
+)
+
+
+@pytest.mark.parametrize('op_courier_exception', [
+    OpCourierBadBundle,
+    OpCourierBadYaml,
+    OpCourierBadArtifact,
+    OpCourierQuayError,
+    OpCourierError
+])
+def test_exception_backwards_compatibility(op_courier_exception):
```

Comment:
Please add docstring that we are testing BW compatiblity

Reply:
Done